lib/matplotlib/projections/geo.py

```diff
@@ -198,27 +225,17 @@ def set_longitude_grid_ends(self, degrees):
             .translate(0.0, -self._longitude_cap)
 
     def get_data_ratio(self):
-        '''
-        Return the aspect ratio of the data itself.
-        '''
+        # docstring inherited
         return 1.0
 
-    ### Interactive panning
+    # Mark interactive panning and zooming as disabled.
 
     def can_zoom(self):
-        """
-        Return *True* if this axes supports the zoom box button functionality.
-
-        This axes object does not support interactive zoom box.
-        """
+        # docstring inherited
```

Comment:
confused about what this docstring inherited means

Reply:
See https://matplotlib.org/devdocs/devel/documenting_mpl.html#inheriting-docstrings